opentelemetry-semantic-conventions/scripts/templates/header_resource.rs

```diff
@@ -7,14 +7,31 @@
 //!
 //! ## Usage
 //!
+//! [`tracing`]:

```

Comment:
I think examples in this repo should show OTel Tracing API, not `tracing` crate macros!

[I agree with the decision](https://github.com/open-telemetry/opentelemetry-rust/issues/1043#issuecomment-1776529519) to specially treat `tracing` in the API/SDK given the importance of tracing crate to Rust ecosystem, but the examples in this repo should be showing components shipped from this repo.

Reply:
OK, lacking a specific suggested example to do, if any, for `tracing`, I've just removed the additional example block. I'll leave it to any follow ups that anyone would like to do for docs.